index.html

```diff
@@ -3562,6 +3562,13 @@ <h3>
 The <a>DID resolver</a> was unable to find the <a>DID document</a>
 resulting from this resolution request.
                         </dd>
+<dt>
+representation-not-supported
```

Comment:
```suggestion
unsupported-representation
```

Don't know if this is better than the one in the original PR... thoughts?

Reply:
it's shorter, I don't have strong feeling one way or the other.

Seems to align better with https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/415

I think technically https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/406

is more accurate, perhaps "representation-not-acceptable" is better?

Reply:
hmm... perhaps we want to align w/ HTTP response codes when possible... this would be one such case, and agree that "representation-not-acceptable" is probably the more technically accurate error.

Thoughts from others? Remember that these codes, once defined, will stick around forever, so bikeshedding at this point is appropriate.

Reply:
No additional thoughts from the group, merging as-is.